addons/website/static/src/js/editor/wysiwyg.js

```diff
@@ -116,6 +104,7 @@ Wysiwyg.include({
             if (this.$('.o_mega_menu').hasClass('show')) {
                 this.snippetsMenu.toggleMegaMenuSnippets(true);
             }
+            this.$editable.off('dblclick', 'a');
```

Comment:
The part using `preventLinkDoubleClick` is to prevent the tooltip on simple click that hints to double click.
This is to remove the double click behavior.
Anyway I just removed that line and conditionned directly the dblclick binding with the `preventLinkDoubleClick` option in web_editor's wysiwyg, makes more sense, the same way it is done for the tooltip.